ui/apps/platform/src/Components/ScannerV4IntegrationBanner.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Banner, Flex, FlexItem } from '@patternfly/react-core';
+import { InfoCircleIcon } from '@patternfly/react-icons';
+
+import useRestQuery from 'hooks/useRestQuery';
+import useMetadata from 'hooks/useMetadata';
+import { fetchImageIntegrations } from 'services/ImageIntegrationsService';
+import { getVersionedDocs } from 'utils/versioning';
+import { scannerV4ImageIntegrationType } from 'types/imageIntegration.proto';
+
+function ScannerV4IntegrationBanner() {
+    const { version } = useMetadata();
+    const { data, loading, error } = useRestQuery(fetchImageIntegrations);
+
+    // Don't show the banner if we don't have successful responses
+    if (!data || loading || error || !version) {
+        return null;
+    }
+
+    const hasScannerV4Integration = data.some(({ type }) => type === scannerV4ImageIntegrationType);
+
+    // Don't show the banner if we have a Scanner V4 integration
+    if (hasScannerV4Integration) {
+        return null;
+    }
+
+    return (
+        <Banner variant="info">
+            <Flex
+                alignItems={{ default: 'alignItemsCenter' }}
+                justifyContent={{ default: 'justifyContentCenter' }}
+            >
+                <FlexItem>
+                    <InfoCircleIcon />
+                </FlexItem>
+                <FlexItem>
+                    New Scanner V4 now generally available in RHACS 4.4. Refer to the{' '}
```

Comment:
Observation, not change request because this is temporary with short deadline: product branding.

Reply:
Oh good catch. Worthwhile to change now IMO. I'll get rid of the icon as well in the next commit.

I'll change the branding for the first instance of "RHACS", but I'll leave "RHACS Documentation" since it will always link to the Red Hat branded documentation page.